### ISSUE TITLE
Investigate missing adoption of Hostinger

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/httparchive.org#617]
Hostinger exists as a supported technology in Wappalyzer, but we're not detecting any pages that use them.

Looking at the Wappalyzer source code, this technology seems to be using an unusual detection method (DNS):

https://github.com/wappalyzer/wappalyzer/blob/c4704aa76175a98d4212bcaa126ceb1473e51e8e/src/technologies/h.json#L888-L902

@pmeenan is that something that is supported by the WPT agent's driver code?

cc @ThierryA

**Comment by pmeenan** [comment]
No, the DNS code path isn't wired up in the agent (and doing it would probably be a fairly big change to the order the agent does things).

WPT does do a DNS pass for the base page though and logs the authoritative DNS for the origin:

base_page_dns_server: "any1.hostinger.com",

As well as a reverse-IP lookup on the origin IP (and any CNAME that the origin uses):

base_page_ip_ptr: "",
base_page_cname: "",

If you're looking for hosting information, that's probably the more reliable way to do it.

Can open a WPT agent issue to do the DNS work before running wappalyzer but I'm not sure it will fly because they are currently done in parallel and it will slow down tests.

**Comment by rviscomi** [comment]
Thanks for looking into it. I suppose it can't hurt to open the issue on the WPT side to at least track the limitation and explore alternatives.

It's gross but I suppose it's possible that we can backstop some of these missing detections on the HA side using that host metadata. For example, in the Dataflow pipeline and test each page against Wappalyzer's DNS rules and emulate the Wappalyzer detections in the HAR.
